spec/components/validators/integer_setting_validator_spec.rb

```diff
@@ -21,9 +21,21 @@
 
       it "returns true if value is a valid integer" do
         expect(validator.valid_value?(1)).to eq(true)
-        expect(validator.valid_value?(-1)).to eq(true)
         expect(validator.valid_value?('1')).to eq(true)
-        expect(validator.valid_value?('-1')).to eq(true)
+      end
+
+      it "defaults min to 0" do
```

Comment:
If the default min would be `5`, I would have replaced:

```
expect(validator.valid_value?(-1)).to eq(false)
```

with

```
expect(validator.valid_value?(4)).to eq(false)
```

> This description is alittle odd for me

What should be the description here?